src/Font.cpp

```diff
@@ -574,9 +574,9 @@ void Font::LoadFontPageSettings( FontPageSettings &cfg, IniFile &ini, const std:
 				int first = 0;
 				if( !asMatches[2].empty() )
 				{
-					sscanf( asMatches[2].c_str(), "%x", &first );
+					sscanf( asMatches[2].c_str(), "%x", (unsigned int *)&first );
```

Comment:
Casing a signed int to an unsigned int when reading into it seems wrong.  The format string should be fixed to use the right type.